apps/testrunner/test_suites/selenium_tests/main_page_selenium_test.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import logging
+from selenium.webdriver.common.keys import Keys
+from common.mw_url_generator import MWUrlGenerator
+from testrunner.test_suites.selenium_tests.selenium_test import SeleniumTest
+
+logger = logging.getLogger(__name__)
+
+class MainPageSeleniumTest(SeleniumTest):
+    def do_run(self, driver, timer):
```

Comment:
The reason is that all common functionality goes to the base SeleniumTest class and specific scenario logic goes to the sub class - it's a strategy design pattern. Right now it doesn't add much, but as a next step I will create more classes like the MainPageSeleniumTest with different selenium scenarios